lib/pure/nativesockets.nim

```diff
@@ -447,7 +447,7 @@ proc getSockDomain*(socket: SocketHandle): Domain =
     raiseOSError(osLastError())
   try:
     result = toKnownDomain(name.sin6_family.cint).get()
-  except UnpackError:
+  except UnpackDefect:
```

Comment:
As @Araq said, I guess you should change this code to something like:
```nim
let res = toKnownDomain(name.sin6_family.cint)
if res.isSome():
  result = res.get()
else:
  raise newException(IOError, "Unknown socket family in getSockDomain")
```

Reply:
It's fixed.